packages/react-components/react-accordion/src/components/Accordion/Accordion.test.tsx

```diff
@@ -8,7 +8,12 @@ describe('Accordion', () => {
     Component: Accordion,
     displayName: 'Accordion',
     // Accordion does not have own styles
-    disabledTests: ['make-styles-overrides-win', 'consistent-callback-args'],
+    disabledTests: ['make-styles-overrides-win', 'consistent-callback-args-legacy'],
+    testOptions: {
+      'consistent-callback-type': {
+        ignoreProps: ['onToggle'],
```

Comment:
The way these conformance tests are set up seems like a configuration headache. It's confusing when the `-legacy` version should be enabled. It's also not obvious why some props are being ignored, and when/if new props should be ignored as well.

IMO we should keep a single `consistent-callback-args` check: don't split it into two tests that have to be enabled/disabled in different scenarios. The test could be configured to check legacy callbacks against the legacy pattern (rather than simply ignored).

E.g.
```suggestion
    disabledTests: ['make-styles-overrides-win'],
    testOptions: {
      'consistent-callback-args': {
        legacyCallbacks: ['onToggle'],
```

The config option could be named `legacyCallbacks_doNotAddNewPropsHere` or something, to make it even clearer.